peripheral.xarcade/resources/language/resource.language.fr_fr/strings.po

```diff
@@ -17,11 +17,12 @@ msgstr ""
 "Plural-Forms: nplurals=2; plural=n > 1;\n"
 "X-Generator: Weblate 4.9\n"
 
-#, fuzzy
 msgctxt "Addon Summary"
 msgid "X-Arcade Tankstick driver"
 msgstr "Pilote Tankstick X-Arcade"
 
 msgctxt "Addon Description"
 msgid "This driver enables input from the X-Arcade Tankstick.[CR][CR]A udev rule is needed. See https://kodi.wiki/view/Tankstick"
 msgstr ""
+
+#, fuzzy
```

Comment:
Wanted?

Reply:
It's created by Weblate and commented out.